lightfm/_lightfm_fast.pyx.template

```diff
@@ -780,6 +780,8 @@ def fit_logistic(CSRMatrix item_features,
                item_alpha,
                user_alpha)
 
+    return loss
```

Comment:
This is, in fact, the loss _of the current example_. So not what you want at all!